### DISCUSSION TITLE
backgroundColor "transparent" transition problem

### DISCUSSION THREAD

**Discussion by magdalko** [pmndrs/react-spring#1623] [Support]
hello, I have a set of buttons which change the background color when hovering or clicking. I found something weird when it comes to transparent background color. It turns it first to grey color and then changing to the demanded color (in this case blue). 
 
Here is an example in codesandbox, where you can see the problem (it is set to transparent now, but change backgroundColor to white where it says transparent in index.js to see the difference): https://codesandbox.io/s/react-spring-hover-forked-w76pw?file=/src/index.js

Here: from bgcolor transparent

https://user-images.githubusercontent.com/31534726/125078116-402e8580-e0c2-11eb-9b83-639744653ff2.mov

Here: from bgcolor white

https://user-images.githubusercontent.com/31534726/125078205-5fc5ae00-e0c2-11eb-90fe-d1d7642852cd.mov

Anyone knows what to do about it?

**Comment by Sh4reef** [comment]
@magdalko I've opened the codesandbox I didn't notice your issue transitioning from `backgroundColor: transparent`

**Comment by joshuaellis** [comment]
this has been answered in #1627
